exasol_transformers_extension/deployment/language_container_deployer.py

```diff
@@ -93,7 +94,14 @@ def run(cls, bucketfs_name: str, bucketfs_host: str, bucketfs_port: int,
             dsn: str, db_user: str, db_password: str, language_alias: str):
 
         pyexasol_conn = pyexasol.connect(
-            dsn=dsn, user=db_user, password=db_password)
+            dsn=dsn,
+            user=db_user,
+            password=db_password,
+            encryption=True,
+            websocket_sslopt={
+                "cert_reqs": ssl.CERT_NONE,
+            }
+        )
```

Comment:
For every functionality which is not test related, I would recommend enable the user of the functionality to decide on enabling/disabling the certificate check.

Reply:
Agree, will do this on another ticket

Reply:
https://github.com/exasol/transformers-extension/issues/103